src/common/options.cc

```diff
@@ -4478,15 +4478,20 @@ std::vector<Option> get_global_options() {
     .add_see_also("bluestore_cache_size"),
 
     Option("bluestore_cache_meta_ratio", Option::TYPE_FLOAT, Option::LEVEL_DEV)
-    .set_default(.4)
+    .set_default(.45)
```

Comment:
what's the motivation behind increasing the meta and kv ratio? cache_data_ratio now ends up becoming 0.06 from 0.2

Reply:
It seems like we've pretty consistently seen that in memory limited scenarios (with rbd) that every bit of memory you can give to the onode cache helps significantly.  Once we have the age-binning code in we can tweak this further to perhaps prioritize very recent data over older onodes at a higher rate, but I think for now we probably want to really aggressively prioritize bluestre onode cache and rocksdb block_cache for omap over data cache and rocksdb block_cache for onode.